docs/src/parameter_distributions.md

```diff
@@ -21,119 +26,106 @@ dict_1 = Dict("distribution" => distribution_1, "constraint" => constraint_1, "n
 dict_2 = Dict("distribution" => distribution_2, "constraint" => constraint_2, "name" => name_2)
 prior = ParameterDistribution( [dict_1, dict_2] )
 ```
+
 We provide many examples in the package `examples/`, the unit tests found in `test/ParameterDistributions/runtests.jl`, and in the documentation below.
 
-# A simple example:
-Task: We wish to create a prior for a one-dimensional parameter. Our problem dictates that this parameter is bounded between 0 and 1. Prior knowledge dictates it is around 0.7. The parameter is called `point_seven`.
+### Constrained\_gaussian constructor
```

Comment:
Perhaps reword? "The `constrained_gaussian` constructor (recommended)"

Reply:
Addressed in [9b09cea](https://github.com/CliMA/EnsembleKalmanProcesses.jl/pull/161/commits/9b09cea7032cefb92cd0332b9656b924152356c1), which implements the recommended reorganization.